### DISCUSSION TITLE
Transition Plan for April 2021

### DISCUSSION THREAD

**Discussion by gingerBill** [odin-lang/Odin#916] [Planning]
Related Issues: 
* Replacing the Old Backend with the New Backend Transition Period #891
* New Odin Versioning System for Pre Version 1.0 #887
* Eventually making -strict-style the default mode #871
* Make the odin root directory configurable through an environment variable #913

Stages:

* [x] Get `-llvm-api` working on macOS and Linux (officially) #912
* [x] Get `-llvm-api` working in the CI and Nightly Builds #912
* [x] Create Release which has support for the old backend and new backend
* [x] Add binaries to pre-dev release
* [x] Update to new versioning system  #887
* [x] Remove the old backend #891 #914
* [x] Make `-strict-style` default #871
* [x] Allow `ODIN_ROOT` environment variable
* [ ] Release of `dev-2021-04` (this month's release with the old backend removed) (scheduled for before April 30th)

**Comment by ThisDevDane** [comment]
#914 Also have the `Update to new versioning system` part?

**Reply by gingerBill** [reply]
Already supported in the new PR
